docs/feature_rgblight.md

```diff
@@ -37,8 +37,8 @@ QMK uses [Hue, Saturation, and Value](https://en.wikipedia.org/wiki/HSL_and_HSV)
 
 <img src="gitbook/images/color-wheel.svg" alt="HSV Color Wheel" width="250"/>
 
-Changing the **Hue** cycles around the circle.  
-Changing the **Saturation** moves between the inner and outer sections of the wheel, affecting the intensity of the color.  
+Changing the **Hue** cycles around the circle.
```

Comment:
These lines should keep the double spaces at the end, having it all on one line doesn't look as good.

Reply:
Sorry editor removed those as extra white space. I'll fix it

Reply:
I've introduced <br> as this is a better solution than space which is really relying on a quirk in markdown to introduce a line break and is invisible.

Reply:
I thought that was a feature of markdown... https://daringfireball.net/projects/markdown/syntax#p

Reply:
don't disagree, but it's hard to tell that from a mistake while editing. Soooo I'd rather just use <br> and remove the ambiguity.